content/docs/command-reference/status.md

```diff
@@ -157,44 +157,45 @@ that.
 
 ```dvc
 $ dvc status
-bar.dvc:
-        changed deps:
-                modified:      bar
-        changed outs:
-                not in cache:      foo
-foo.dvc:
-        changed outs:
-                deleted:      foo
-        changed checksum
-prepare.dvc:
-        changed outs:
-                new:      bar
-        always changed
+dofoo:
+	changed deps:
+		modified:           bar
+	changed outs:
+		not in cache:       foo
+dobar:
+	changed outs:
+		deleted:            bar
```

Comment:
But now we need to worry about the consistency of the example. How come dofoo comes first but it uses bar as a dependency? bar is the output of dobar which comes after...

Please try to actually codify these examples in a dvc.yaml file and run repro/status as needed so they make sense.

Reply:
changing deps of dofoo to `baz` make it is consistent. It will become a simple pipeline:
```
+---------+  
| baz.dvc |  
+---------+  
      *      
      *      
      *      
 +-------+   
 | dofoo |   
 +-------+   
      *      
      *      
      *      
 +-------+   
 | dobar |   
 +-------+ 
```

Reply:
Better! But the current example doesn't show a connection between dofoo and dobar. In fact dobar is a separate pipeline altogether as it has no dependencies.

Reply:
Have you actually codified it in dvc.yaml so we can see what the output would be? I'm not sure about how `status` orders stages, maybe it's chronological, maybe it's alphabetical?

Reply:
Yes I actually made this pipeline. The output of `status` was correct as it was after executing `dofoo` stage with `--no-commit`. In the new update it shows connection between stages.

Reply:
So it prings `baz.dvc` at the end? Hmmm interesting... OK

Thanks

Reply:
p.s. @imhardikj can you please share the dvc.yaml with me so I can confirm about the order? I may open a ticket in the core repo about this...

Reply:
This was `dvc.yaml`.
```
$ cat dvc.yaml
stages:
  dofoo:
    cmd: echo "Hello" > foo
    deps:
    - baz
    outs:
    - foo
  dobar:
    cmd: echo "Hello" > bar
    deps:
    - foo
    outs:
    - bar
```

Reply:
> So it prints baz.dvc at the end

Most of times `baz.dvc` was in starting. If you check in commit https://github.com/iterative/dvc.org/pull/1952/commits/f0ff0ee229992af3fdfbd74790af8a06687b6176, I moved `baz.dvc` to starting. Here in this review https://github.com/iterative/dvc.org/pull/1952#discussion_r534370722, `output of status was correct` was actually meant for `baz.dvc` at starting.
But in the next commit https://github.com/iterative/dvc.org/pull/1952/commits/b54bf7bef8ec6171cb91d42a4cff44ec63946fec I moved it back to last because I was testing this got this output once, so I thought this was also correct. But yes majority of times `baz.dvc` was in starting only that's why I changed it in https://github.com/iterative/dvc.org/pull/1952/commits/f0ff0ee229992af3fdfbd74790af8a06687b6176.

Reply:
It should be in the beginning. I'm glad I double checked this... I'll fix it later.

Reply:
Can I open the PR for this. There's one more example remaining in `fetch` which requires indentation change. I removed it from this PR as it was showing merge conflict. https://github.com/iterative/dvc.org/pull/1952#issuecomment-732238929

Reply:
I have the change stashed locally already, so it will get to the repo eventually. But feel free to send a PR, sure. No big deal either way.

Reply:
p.s. don't shy away from merge conflicts 🙂